db/seeds/02_data.rb

```diff
@@ -1,3 +1,9 @@
+puts 'Destroy previous users ... '
```

Comment:
This looks like a mis-merge between what I added last night - the user-seeding part of this belongs only in `01_users.rb`